ballerina-tests/tests/hateoas_runtime_error_tests.bal

```diff
@@ -0,0 +1,132 @@
+import ballerina/http;
```

Comment:
Lets add licence header

Reply:
Fixed in [9b1a15c](https://github.com/ballerina-platform/module-ballerina-http/pull/1068/commits/9b1a15ce791c2179473d67578d1acbb8c934d313)